### ISSUE TITLE
sample decode lost frames

### ISSUE THREAD

**Issue by mamingy2** [oneapi-src/oneVPL-intel-gpu#268]

**Comment by Jexu** [comment]
Do you check if the lost frames are non-display frame since these frames will not have output yuv. @wangyan-intel can correct me  .

**Comment by mamingy2** [comment]
Yes, the lost frames are not non-display frames. Other decoders can decode without loss.

**Comment by Jexu** [comment]
Ok, therefore we may need to check if this bitstream has violation that vpl may skip decoding some error frames.

**Comment by brmarkus** [comment]
@Jexu can you clarify where to find whether a frame is marked as a non-display frame?
We checked various players, we checked with VQA/Video Codec Analyzer/VQ Analyzer - different tools and players were able to decode and output and render raw YUV/NV12.
But there might be a flag somewhere we and other players&tools overlooked...

**Comment by brmarkus** [comment]
> Ok, therefore we may need to check if this bitstream has violation that vpl may skip decoding some error frames.

We checked with the `corrupted` flag (like modifying `sample_decode` to print each frame's `corrupted` flag), but no bitmask indicated for a decoder error.
See also our feedback on the MediaSDK-github issue created: "https://github.com/Intel-Media-SDK/MediaSDK/issues/3007#issuecomment-1409820983".

**Comment by intel-mediadev** [comment]
Auto Created VSMGWL-63156 for further analysis.
